### ISSUE TITLE
add the given_name, family_name and name to the X-Auth headers

### ISSUE THREAD

**Issue by gambol99** [gambol99/keycloak-proxy#57]
```json
 "resource_access": {},
  "name": "Rohith Jayawardene",
  "preferred_username": "rohith.jayawardene",
  "given_name": "Rohith",
  "family_name": "Jayawardene",
  "email": "gambol99@gmail.com"
```

These are passed in the JWT token as above

**Comment by gambol99** [comment]
would be nice allow customization on this rather hardcode the headers / claims ... i..e --add-claim <name> etc

**Comment by gambol99** [comment]
released in PR #58
